website/content/docs/k8s/installation/vault/wan-federation.mdx

```diff
@@ -0,0 +1,678 @@
+---
+layout: docs
+page_title: Federation Between Kubernetes Clusters with Vault as Secrets Backend
```

Comment:
I originally titled this WAN Federation and that is why it is that way in the docs as well as the file name of`wan-federation.mdx`.   I changed the title since this is essentially a sister document to the existing mesh fed page which is called `Federation Between Kubernetes Clusters`.

We should align on the following:
- page title
- side nav title
- file name / url

having those match as much as possible will help SEO scores.